src/training/buff.rs

```diff
@@ -233,6 +235,23 @@ unsafe fn buff_sepiroth(module_accessor: &mut app::BattleObjectModuleAccessor) -
     false
 }
 
+
+unsafe fn buff_wario(module_accessor: &mut app::BattleObjectModuleAccessor) -> bool {
+    if !is_buffing(module_accessor) {
+        let waft_count: BuffOption = MENU.buff_state.wario_buffs().get_random();
+        if waft_count == BuffOption::empty() {
+            return true
+        }
+        WorkModule::set_int(
+            module_accessor,
+            waft_count.into_int().unwrap(),
+            *FIGHTER_WARIO_INSTANCE_WORK_ID_INT_GASS_COUNT,
```

Comment:
Do we also need to set `GASS_LEVEL`? @GradualSyrup thinks so.

Reply:
yea i played around with it, setting that flag by itself does nothing since it gets overwriiten to normal value the next frame. 

i think whats going on is that the gass_count is being updated and checked every frame and once it reaches mini/half/full waft threshold, the game sets that flag automatically. ill do some output some debug logging to be sure tho.

Reply:
yep, just confirmed the game will set that flag automatically the next frame. i guess if you want to be super safe you can set the gass_level flag on the same frame manually but I honestly dont think itll make a difference.